plugins/poll/assets/javascripts/components/poll-voters.js.es6

```diff
@@ -1,13 +1,49 @@
+import { on } from 'ember-addons/ember-computed-decorators';
+
 export default Ember.Component.extend({
   layoutName: "components/poll-voters",
   tagName: 'ul',
   classNames: ["poll-voters-list"],
   isExpanded: false,
-  numOfVotersToShow: 20,
+  numOfVotersToShow: 0,
+  offset: 0,
+  loading: false,
+  pollsVoters: [],
```

Comment:
Initialzing a value to an array like this is probably not doing what you'd expect: Every instance of the `Component` class will have the same array!

In your case you are replacing it after the ajax call completes so bugs are unlikely, but it's a footgun and I think you should replace it with `pollsVoters: null`. If `pollsVoters` is only used in a template ember should deal with `null` safely, but if you need it to be an empty array put it in an `init()` function.